docs/node-mixin/config.libsonnet

```diff
@@ -17,6 +17,19 @@
     // them here, e.g. 'device!="tmpfs"'.
     diskDeviceSelector: '',
 
+    // Some of the alerts are meant to fire if a critical failure of a
+    // node is imminent (e.g. the disk is about to run full). In a
+    // true “cloud native” setup, failures of a single node should be
+    // tolerated. Hence, even imminent failure of a single node is no
+    // reason to create a paging alert. However, in practice there are
+    // still many situations where operators like to get paged in time
+    // before a node runs out of disk space. nodeCriticalSeverity can
+    // be set to the desired severity for this kind of alerts. This
+    // can even be templated to depend on labels of the node, e.g. you
+    // could make this critical for traditional database masters but
+    // just a warning for K8s nodes.
+    nodeCriticalSeverity: 'critical',
```

Comment:
Rather than introducing ever more growing number of variables we should provide easy to follow examples for overriding anything within alerts.

Reply:
I thought about that, too, but in this case, we just want to set the severity of a specific subset of alerts. Those have to be marked somehow to make them overridable specifically, and I ended up with an additional variable as the best I could come up with.

Reply:
If you have a better idea, let me know.

Reply:
Could you do them as two sets of alerts? That way you could override a chunk of them all together.

I'd also say that the default should be not critical.

Reply:
The two sets of alerts would create the problem that closely related alerts ("disk full in one day" and "disk full in 4h") are then kept in completely different tuples.

I went for "critical" as the default because I expect naive users to be more likely in a situation where single-node failures are problematic, while advanced users with sophisticated setups will have no trouble adjusting the config.

Reply:
I like it most the way I have done it here, but I don't feel strongly.

@metalmatze do you have other ideas how to do this or would you prefer any of the already suggested ideas? If you prefer a separate tuple for this class of alerts and/or `warning` as default, I'm happy to follow the quorum.

Reply:
@metalmatze *ping*

Reply:
Thanks for the ping. I don't have super strong opinions on this particular case. So we can go ahead and merge this. 
Why I pointed this out in the first place was to discuss if we should use composition to override certain groups of alerts, rather than introduce another set of configuration knobs. Nothing we need to discuss in more details here though.
LGTM

Reply:
Alright. Will submit as is. @brian-brazil feel free to create another PR to put your suggestion up for review/discussion.